plone/schemaeditor/browser/schema/schemaeditor.js

```diff
@@ -161,14 +160,14 @@
         });
         // reorder fields and change fieldsets
         $('.fieldPreview.orderable').plone_schemaeditor_html5_sortable(function (position, fieldset_index) {
-            var url = window.location.href.replace('/@@fields', '') + '/' + this.attr('data-field_id') + '/@@order';
+            var url = window.location.pathname.replace('/@@fields', '') + '/' + this.attr('data-field_id') + '/@@order';
```

Comment:
@vangheem in backporting your mockup compatibility fixes to plone.schemaeditor javascript to 1.3.x, I had some issues with reorder ajax URL construction when a querystring containing authenticator token was present in window.location.href.  This change fixes my issues in Plone 4, but should I be forward-porting such a fix to plone.schemaeditor master?